test/truffle-fixture.js

```diff
@@ -0,0 +1,198 @@
+// Libs
```

Comment:
truffle-fixture.js mirrors the `./migrations/2_deploy.js`. I still keep the `./migrations/2_deploy.js` in case we need to deploy something. I'll move to custom deploy functions and completely remove `./migrations/` this week so that we have the same deploy process in testing and in production.